src/models/target/target_D_hybrid.jl

```diff
@@ -117,6 +116,7 @@ function ∂Diffusivity∂θ(
         point in the glacier. Slower but more precise.
         """
         for i in axes(H̄, 1), j in axes(H̄, 2)
+            # We don't use apply_law! because we want to evaluate with custom inputs
```

Comment:
I think that instead of using the function internally like this, it could be a good idea to create a new `apply_law` function with multiple dispatch to handle the type of custom inputs you want. It would definitely be cleaner.